pandas/core/groupby/groupby.py

```diff
@@ -444,14 +444,15 @@ class providing the base-class of operations.
     a `(callable, data_keyword)` tuple where `data_keyword` is a
     string indicating the keyword of `callable` that expects the
     %(klass)s object.
-args : iterable, optional
+*args : iterable, optional
        Positional arguments passed into `func`.
-kwargs : dict, optional
+**kwargs : dict, optional
          A dictionary of keyword arguments passed into `func`.
 
 Returns
 -------
-the return type of `func`.
+object
+    The return type of `func`.
```

Comment:
This should explain what is being returned. I think it'll be clearer if we say something like (you can surely improve my description, just for you to get the idea):

```
The original object with the function `func` applied.
```

Reply:
Great feedback! Updating this PR.